coordinator/src/orderbook/async_match.rs

```diff
@@ -101,6 +101,9 @@ async fn process_pending_match(
         tracing::debug!(%trader_id, order_id=%order.id, "Notifying trader about pending match");
 
         let matches = matches::get_matches_by_order_id(&mut conn, order.id)?;
+
+        // TODO: get the matching fee as a sum from all matches.
```

Comment:
Why do you need the order matching fee here?

Reply:
I think this was outdated. Thanks